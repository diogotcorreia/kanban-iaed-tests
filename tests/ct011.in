u diogo
t 1 lavar os olhos
a WASHING
m 1 diogo WASHING
m 1 diogo IN PROGRESS
m 1 diogo TO DO
m 1 diogo WASHING
m 1 diogo WASHING
m 1 diogo DONE
m 1 diogo IN PROGRESS
n 1
m 1 diogo DONE
q