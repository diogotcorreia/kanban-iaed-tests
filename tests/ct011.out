task 1
task already started
duration=0 slack=-1
1
duration=1 slack=0
